### PR TITLE
Reformat 10 Recipes #2

### PR DESCRIPTION
**Pull request by dash102**
GCI Task: https://codein.withgoogle.com/dashboard/task-instances/5709428917534720/?sp-page=1

### CODE REVIEW

File: app-text/libpaper/libpaper-1.1.24.recipe [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+DESCRIPTION="The libpaper paper-handling library automates recognition of many different \
+paper types and sizes for programs that need to deal with printed output."
 HOMEPAGE="http://packages.debian.org/unstable/source/libpaper"
-SOURCE_URI="http://ftp.de.debian.org/debian/pool/main/libp/libpaper/libpaper_1.1.24.tar.gz"
-CHECKSUM_SHA256="38ac28114673d4b175a939d981e0477a0ed9bcaab27033522ecb2ca78ee90c7e"
-LICENSE="GNU GPL v2"
 COPYRIGHT="1996 Yves Arrouye
-	2000 Adrian Bunk"
-
+	   2000 Adrian Bunk"
```
Comment (waddlesplash) [inline app-text/libpaper/libpaper-1.1.24.recipe]: Only indent by one tab.

File: dev-libs/libedit/libedit-2014_10_30_3.1.recipe [hunk 2]
```diff
@@ -24,6 +23,15 @@ REQUIRES="
...
+PROVIDES_devel="
+	libedit${secondaryArchSuffix}_devel = $portVersion
+	devel:libedit$secondaryArchSuffix = 0.0.52 compat >= 0
+	"
+REQUIRES_devel="
+	libedit$secondaryArchSuffix == $portVersion base
+	"
+
 BUILD_REQUIRES="
 	devel:libncurses$secondaryArchSuffix
```
Comment (humdingerb) [inline dev-libs/libedit/libedit-2014_10_30_3.1.recipe]: Move haiku${secondaryArchSuffix}_devel from BUILD_PREREQUIRES as first entry under BUILD_REQUIRES.

File: media-libs/faad2/faad2-2.7.recipe [hunk 3]
```diff
@@ -25,6 +23,16 @@ PROVIDES="
...
+	faad2${secondaryArchSuffix}_devel = $portVersion compat >= 2.0
+	devel:libfaad$secondaryArchSuffix = 2.0.0 compat >= 2
+	devel:libmp4ff$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	faad2$secondaryArchSuffix == $portVersion base
+	"
+
 BUILD_PREREQUIRES="
 	haiku${secondaryArchSuffix}_devel
```
Comment (humdingerb) [inline media-libs/faad2/faad2-2.7.recipe]: Add a BUILD_REQUIRES and move haiku${secondaryArchSuffix}_devel from BUILD_PREREQUIRES to BUILD_REQUIRES.

File: media-libs/libebur128/libebur128-1.0.1.recipe [hunk 4]
```diff
@@ -1,15 +1,14 @@
 SUMMARY="A library implementing the EBU R128 loudness standard"
-DESCRIPTION="
-libebur128 is a library that implements the EBU R 128 standard for loudness \
-normalisation.
-"
+DESCRIPTION="libebur128 is a library that implements the EBU R 128 standard for loudness \
+normalisation. It supports all samplerates by recalculation of the filter coefficients"
```
Comment (humdingerb) [inline media-libs/libebur128/libebur128-1.0.1.recipe]: Add a "." at the end.

File: media-libs/libebur128/libebur128-1.0.1.recipe [hunk 5]
```diff
@@ -22,6 +21,14 @@ REQUIRES="
...
+	libebur128${secondaryArchSuffix}_devel = $portVersion
+	devel:libebur128$secondaryArchSuffix = 1.0.1 compat >= 1
+	"
+REQUIRES_devel="
+	libebur128$secondaryArchSuffix == $portVersion
+	"
+
 BUILD_REQUIRES="
 	"
 #	devel:libspeexdsp$secondaryArchSuffix
```
Comment (humdingerb) [inline media-libs/libebur128/libebur128-1.0.1.recipe]: Move line 34 under BUILD_REQUIRES.

File: media-libs/libsdl2/libsdl2-2.0.1.recipe [hunk 6]
```diff
@@ -41,7 +53,6 @@ BUILD_REQUIRES="
 	devel:libjpeg$secondaryArchSuffix
 	"
 #	devel:libtiff$secondaryArchSuffix # haiku_devel missing devel_libtiff
```
Comment (humdingerb) [inline media-libs/libsdl2/libsdl2-2.0.1.recipe]: Move line 34 under BUILD_REQUIRES (before the closing ").

File: net-libs/libircclient/libircclient-1.8.recipe [hunk 7]
```diff
@@ -1,20 +1,19 @@
 SUMMARY="An IRC client library"
-DESCRIPTION="
-libircclient is a library implementing the client-server IRC protocol.
-"
+DESCRIPTION="libircclient is a library implementing the client-server IRC protocol. \
+It is designed to be small, fast, portable and compatible with the RFC standards \
+as well as non-standard but popular features. \
```
Comment (humdingerb) [inline net-libs/libircclient/libircclient-1.8.recipe]: Remove the " \" after "features.". This is supposed to be a paragraph, i.e. a newline is intended.

File: app-text/libpaper/libpaper-1.1.24.recipe [hunk 8]
```diff
@@ -22,15 +19,22 @@ PROVIDES="
...
 REQUIRES="
 	haiku$secondaryArchSuffix
 	"
 
+PROVIDES_devel="
+	libpaper${secondaryArchSuffix}_devel = $portVersion
+	devel:libpaper$secondaryArchSuffix = 1.1.24 compat >= 1
+	"
+REQUIRES_devel="
+#	haiku${secondaryArchSuffix}_devel
```
Comment (humdingerb) [inline app-text/libpaper/libpaper-1.1.24.recipe]: You can remove this line completely.

**Comment by dash102** [comment]
I have fixed the commented parts.

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**